variables.json

```diff
@@ -1,16 +1,16 @@
 {
   "AIRFLOW_DATA_BUCKET": "tulib-airflow-prod",
-  "DPLAH_TARGET_ALIAS_ENV": "dev",
-  "DPLAH_XSL_CONFIG": {
+  "DPLAH_HARVEST_CONFIG": {
```

Comment:
my mistake, remove the harvest config variable. We need to maintain this variable manually in the UI so that the username/password isn't exposed in github